yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -423,7 +423,6 @@ func NewPrimaryOrUniqueConsOnUnsupportedIndexTypesIssue(objectType string, objec
 		CONSTRAINT_NAME: constraintName,
 	}
 	issue := primaryOrUniqueOnUnsupportedIndexTypesIssue
-	issue.Name = fmt.Sprintf(issue.Name, typeName)
 	issue.Description = fmt.Sprintf(issue.Description, constraintName)
```

Comment:
why are we setting `constraintName` with description? it should `typeName` I believe?

Reply:
yes, i had fixed that in later commits when tests failed.